patternfly/patternfly-elements
# fix pfe-card's published files

---

### What has changed and why

pfe-card had two `files` properties in its package.json, which led to no files being published.  This PR corrects that.

### Testing instructions

```
cd elements/pfe-card
npm publish --dry-run
```

Verify in the output that "Tarball Contents" lists files in the `dist` directory, like so:

```
npm notice === Tarball Contents ===
npm notice 1.4kB  package.json
npm notice 1.1kB  LICENSE.txt
npm notice 4.6kB  README.md
npm notice 196B   dist/pfe-card--lightdom.css
npm notice 16.8kB dist/pfe-card--lightdom.css.map
npm notice 180B   dist/pfe-card--lightdom.min.css
npm notice 202B   dist/pfe-card--lightdom.min.css.map
npm notice 19.2kB dist/pfe-card.js
npm notice 24.5kB dist/pfe-card.js.map
npm notice 16.6kB dist/pfe-card.min.js
npm notice 22.9kB dist/pfe-card.min.js.map
npm notice 23.7kB dist/pfe-card.umd.js
npm notice 24.3kB dist/pfe-card.umd.js.map
npm notice 17.3kB dist/pfe-card.umd.min.js
npm notice 22.3kB dist/pfe-card.umd.min.js.map
```

Before the fix, no `dist` files were included:

```
npm notice === Tarball Contents ===
npm notice 1.4kB  package.json
npm notice 1.1kB  LICENSE.txt
npm notice 4.6kB  README.md
```

### Ready-for-merge Checklist
